### ISSUE TITLE
Balance suggestions and other feedback

### ISSUE THREAD

**Issue by SpencerWS** [JohnnyBazooka89/StSModTheBlackbeard#5]
I've made some videos showcasing this mod, and here is a thorough review of its cards. Headphones and increasing the playback speed will help in watching this video; sorry about that, I'm new to making good videos. https://youtu.be/8ZFrvxDpfMg

Some comments/suggestions not made in the video:
-Make weapons more interesting by letting them give resistance. Rare Halbard could be 4/3 with 2 resistance, for example.
-Currently weapon bonuses are not multiplied by pen nib (Are they multiplied by vulnerable?). That makes weapons of less utility than strength.
-Bug report: Writhing mass will crash game by putting curse in a modded character's deck.

**Comment by JohnnyBazooka89** [comment]
Thank you for your opinions and for the video you made. I don't necessarily agree with many of your points and your evaluations of cards, but I would have to answer to every card individually.

I certainly heard a few times, that Weapons are not good/not worth playing. I think atm it is possible to play with Weapons in a few different ways (dagger autoplay build/cat of o' nine tails build etc). I certainly try to make them interesting (I just recently added 2 new Weapon relics), but I know that they may be a little slow/setup heavy, especially when playing on high ascensions.

(This is an example of me playing with Cat O' Nine Tails build: xD)
![unknown](https://user-images.githubusercontent.com/43926043/50337779-701cda00-0512-11e9-9518-b6355764ea9c.png)
![unknown2](https://user-images.githubusercontent.com/43926043/50338035-4912d800-0513-11e9-9d08-6015b9ffbbc3.png)

About the comments not made in the video:
- I like them being separate. I could add one Weapon that gives you resistance, but one problem is that you would probably never want to end durability of this Weapon to keep resistance for longer.
- In my tests both Pen Nib (also my relic Penknife), and Vulnerable worked properly with Weapons. I couldn't reproduce this issue.
- In my test I just got Parasite as expected. I couldn't reproduce this issue as well.
